recipe/meta.yaml

```diff
@@ -64,3 +63,4 @@ about:
 extra:
   recipe-maintainers:
     - beckermr
+    - aldanor
```

Comment:
I am v. excited to have another maintainer! Thank you so much! I cannot add you to the repo however. :( I am pinging @jakirkham from the conda-forge team to see if he can do it. 

@jakirkham, can we add @aldanor as a maintainer on this feedstock?

Reply:
IIRC I will be added automatically as a maintainer once this PR is merged in.

Reply:
Ahhh cool! :)

Reply:
> ...I will be added automatically as a maintainer once this PR is merged in.

That's correct. Just to make sure, you do have merge permissions @beckermr, correct?

Reply:
Yep. I can merge! Thanks @jakirkham and sorry to bother you with this!

Reply:
np. We have been trying to make the team maintenance more streamlined and responsive. So things have likely changed a lot since we last chatted about them. If for some reason @aldanor is not added after merging, you can use [`@conda-forge-admin, please update team`]( https://conda-forge.org/docs/webservice.html#conda-forge-admin-please-update-team ) in a new issue to trigger a team update on the feedstock.